app/controllers/users_controller.rb

```diff
@@ -121,7 +124,7 @@ def create # rubocop:disable all
       session.delete(:new_username) if @cas_auth
       flash[:notice] = 'Successfully created user.'
       # log in the new user
-      sign_in @user, bypass: true unless current_user.present?
+      bypass_sign_in @user if current_user.blank?
```

Comment:
Shouldn't this be `.nil?` instead of `.blank?`? I assume the result is the same but blank feels weird when referring to an object as opposed to a string.